### ISSUE TITLE
Julia 0.6.0 not available on Windows

### ISSUE THREAD

**Issue by ufechner7** [JuliaLang/juliaup#1132]
On Linux I can do:
```
juliup add 0.6.0
```
This does not work on Windows:
```
juliaup add 0.6.0
Error: '0.6.0' is not a valid Julia version or channel name.
```

It would be better if older Julia versions would also be available on Windows to be able to make old Julia programs work before starting to port them to newer Julia versions.

**Comment by ViralBShah** [comment]
I believe this is because we do not have tar.gz/zip installers on windows prior to 1.0

**Comment by DilumAluthge** [comment]
Yeah. For Julia versions prior to 1.0, you can download them from the julialang.org website.

Juliaup only supports Julia >= 1.0. (The fact that it might work for some pre-1.0 versions on some platforms is just by happenstance.)
